src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,85 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_SIZE = 1 << 4;
```

Comment:
```suggestion
    private static final int INITIAL_SIZE = 16;
```
IMO it only decreases readability